README.md

```diff
@@ -166,7 +166,7 @@ Note: It is not called if _open_ is changed by other means. Passes through to th
   ```
   see: https://github.com/moroshko/react-autosuggest#rendersuggestion-required.
 
-  Note: the _suggestion.hightlight_ will contain the rendered markup from [fuzzysort](farzher/fuzzysort#fuzzysorthighlightresult-openb-closeb), see the ```options``` prop. If the ```options``` prop contains an array of "keys" then then _suggestion.hightlight_ will contain an array of matches, see: [fuzzysort advanced usage](https://github.com/farzher/fuzzysort#advanced-usage) or checkout the [sampleChromeCommand.js](examples/sampleChromeCommand.js)
+  Note: the _suggestion.highlight_ will contain the rendered markup from [fuzzysort](farzher/fuzzysort#fuzzysorthighlightresult-openb-closeb), see the ```options``` prop. If the ```options``` prop contains an array of "keys" then then _suggestion.hightight_ will contain an array of matches, see: [fuzzysort advanced usage](https://github.com/farzher/fuzzysort#advanced-usage) or checkout the [sampleChromeCommand.js](examples/sampleChromeCommand.js)
```

Comment:
Looks like we both missed one. Want to update the second “hightight”? I’ll be happy to merge after

Reply:
Ah oops, missed that second one. Changed now.